src/Catalyst.Abstractions/Kvm/IEthRpcService.cs

```diff
@@ -0,0 +1,53 @@
+using Catalyst.Abstractions.Kvm.Models;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Catalyst.Abstractions.Kvm
+{
+    public interface IEthRpcService
+    {
+        //long eth_chainId();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4411902)